### ISSUE TITLE
Survey user satisfaction with the avatars feature

### ISSUE THREAD

**Issue by rfk** [mozilla/fxa-content-server#3010]
We need a way to measure whether users like our avatars feature.  Channeling ideas from the recent metrics brantina [1] and a recent discussion with @vladikoff: what if we just asked them directly? 
 
There's lot to figure out about exactly whether and how we could do this a respectful manner, but we should try.  At about 41:00 in the linked airmo video is an interesting discussion of some options.  The folks behind input.mozilla.org may have some good starting points for us as well.

Thoughts?  /cc @vladikoff @ryanfeeley 

[1] https://air.mozilla.org/august-brantina-with-hiten-shah-founder-kissmetrics-crazy-egg/

**Comment by vladikoff** [comment]
Here's a draft (sorry it uses the old settings design):

![image](https://cloud.githubusercontent.com/assets/128755/9523502/59ae1b4c-4ca8-11e5-860c-bf66c79bdcdb.png)

**Comment by ckarlof** [comment]
I love the penguin. 

I feel we need to think a little bit about where's the most appropriate to solicit the user's feedback. It's not clear that our FxA settings page is the best place because the avatar isn't necessarily adding value there. It might be better to solicit feedback after the user gets to see it in action, e.g., after she sees it in her browser chrome.

Focusing exclusively on avatars might be also be the wrong thing. For example, the right thing to do might be to get feedback on "signing into the browser" (or sync). In that feedback process, we could ask the user what she likes best about signing in and one of those could be "personalizing my browser with a profile image". 

I think it's worth taking a look our current mechanisms in Firefox for gathering feedback, e.g., https://wiki.mozilla.org/Firefox/Input/Heartbeat. I think this just gathers generally feedback about Firefox, but I believe it could also be used to gather feature specific feedback in Firefox as well.

**Comment by ryanfeeley** [comment]
I appreciate that a lot of engineering work went in to avatars, but it's a table stake feature, and a very simple one, and I don't think users have meaningful feedback to give about it.

Synced tabs, for example, is a very meaty feature that they can sink their teeth in to, and develop a strong opinion about.

**Comment by rfk** [comment]
> https://wiki.mozilla.org/Firefox/Input/Heartbeat.

Aha, thanks!  I was trying to remembr the name of that thing, but couldn't, and had to settle for a handy-wavy reference to input.mozilla.org.

> it's a table stake feature

Tangent: I've noticed us saying this more and more lately (not meaning to call out @ryanfeeley specifically - I've noticed *myself* saying it more) but I'm not sure it's a useful distinction.  It seems to be shorthand for "we don't need to ask hard questions about the value of the feature", but how do we decide if something is table-stakes or not unless we ask and answer those questions?
